### PR TITLE
Script for verifying country names against WCS

### PR DESCRIPTION
**Pull request by billy-reilly**
It highlights countries that have been set up in WCS which do not have a corresponding entry in this fork where the names are exactly the same. It's important that the names are exactly equal as we are using this package to generate the delivery country names in checkout, and if they do not match orders will fail.

example output:
```bash
▶ npm run check-countries

> country-list@2.1.0 check-countries /Users/reillyb/Documents/www/country-list
> node ./scripts/check-names-match

📤 fetching complete list of countries configured for delivery in WCS...
❌ bad contentType from dpfr - text/plain
❌ bad contentType from dpde - text/plain
❌ bad contentType from evde - text/plain
❌ bad contentType from wlus - text/plain


📋 comparing WCS country names with our country-list...
✅ Albania
✅ Andorra
✅ Anguilla
✅ Argentina
✅ Armenia
✅ Australia
✅ Austria
✅ Azerbaijan
✅ Bahamas
✅ Bangladesh
✅ Barbados
✅ Belgium
✅ Bermuda
✅ Bolivia
✅ Bosnia and Herzegovina
✅ Brazil
✅ Brunei
✅ Bulgaria
✅ Cambodia
✅ Canada
✅ Cayman Islands
✅ Central African Republic
✅ Chile
✅ China
✅ Colombia
✅ Cook Islands
✅ Croatia
✅ Czech Republic
✅ Denmark
✅ Djibouti
✅ Eritrea
✅ Estonia
✅ Falkland Islands
✅ Faroe Islands
✅ Finland
✅ France
✅ French Guiana
✅ French Southern Territories
✅ Gambia
✅ Germany
✅ Greece
✅ Greenland
✅ Guadeloupe
✅ Guernsey
✅ Guyana
✅ Holy See
✅ Hong Kong S.A.R. of China
✅ Hungary
✅ India
✅ Indonesia
✅ Ireland
✅ Italy
✅ Jamaica
✅ Japan
✅ Jersey
✅ Kazakhstan
✅ Kenya
✅ Kiribati
❌ Korea South
✅ Kyrgyzstan
✅ Latvia
✅ Liechtenstein
✅ Lithuania
✅ Luxembourg
❌ Macau S.A.R. of China
✅ Madagascar
✅ Malaysia
✅ Maldives
✅ Marshall Islands
✅ Mexico
✅ Moldova
✅ Monaco
✅ Montenegro
✅ Morocco
✅ Nepal
✅ Netherlands
✅ New Caledonia
✅ New Zealand
✅ Niger
✅ Norway
✅ Oman
✅ Pakistan
✅ Palau
✅ Panama
✅ Philippines
✅ Poland
✅ Portugal
✅ Puerto Rico
✅ Romania
✅ S Georgia & S Sandwich Islands
✅ Saint Kitts and Nevis
✅ Saint Lucia
✅ Saint Pierre and Miquelon
❌ Saint Vincent and the Grenadines
✅ Samoa
✅ San Marino
✅ Sao Tome and Principe
✅ Serbia
✅ Seychelles
✅ Singapore
✅ Slovakia
✅ Solomon Islands
✅ South Africa
❌ South Georgia and the South Sandwich Islands
✅ Spain
✅ Sri Lanka
✅ St Vincent and the Grenadines
✅ Suriname
✅ Svalbard and Jan Mayen
✅ Sweden
✅ Switzerland
✅ Taiwan
✅ Thailand
✅ Togo
✅ Tunisia
✅ Turkey
✅ Turkmenistan
✅ Tuvalu
✅ Ukraine
✅ United Kingdom
✅ United States
✅ Uruguay
✅ Vanuatu
✅ Vietnam
❌ Virgin Islands British
```

### CODE REVIEW

File: scripts/check-names-match.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
+#! usr/bin/env node
+
+const fs = require("fs");
```
Comment (c0ndu17) [inline scripts/check-names-match.js]: Could you use fs-extra rather, it just simplifies file handling.

https://github.com/jprichardson/node-fs-extra

File: scripts/check-names-match.js [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+const https = require("https");
+const qs = require("querystring");
+const url = require("url");
+
+const env = process.argv[2] || "prod";
```
Comment (c0ndu17) [inline scripts/check-names-match.js]: FYI, minimist is a great package for handling cli arguments. Don't need to change this, just a heads up

File: scripts/check-names-match.js [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+      }/webapp/wcs/stores/servlet/SiteOptions?${query}`;
+      const headers = { monty: "true" };
+
+      const { protocol, hostname, path } = url.parse(requestURL);
+      const options = { protocol, hostname, path, headers };
+
+      promises.push(
+        new Promise(resolve => {
+          let rawData = "";
+          https
```
Comment (c0ndu17) [inline scripts/check-names-match.js]: Could we also please use superagent, not a big fan of handling/appending to buffers

Comment (c0ndu17) [reply scripts/check-names-match.js]: Also could we not create a few functions here, it'll make the code a bit more readable. i.e. makeRequest, handleSuccess, handleFailure

**Comment by billy-reilly** [comment]
@c0ndu17 updated

**Pull request merged**